Add Custom Event in CustomTwilioVideoView

jagamypriera opened the pull request:
## Introduction

The type of this PR is: Enhancement (improve an implementation, refactor, chores)



- This PR is related to #[3524](https://github.com/senseobservationsystems/goalie-2-mobile-app/pull/3524) <!-- issue-nr(s) -->

## Description
This PR fix crash when doing video call by using custom event for CustomTwilioVideoView.

## Test Plan
- Build debug/production.
- Do a video call.
- App should not crashed when call connected.

## Checklist
<!--
Put an `x` in the boxes that apply.
Use `~~` around the checklist items that are not applicable to this PR.
Examples:
- [x] I've done this
- ~~[ ] I don't need this~~
-->
- [ ] Code quality is high (good comments, no unneeded changes, good names)
- [ ] Translation keys are used and added to the [spreadsheet](https://docs.google.com/spreadsheets/d/1-6DFzonKzuv_jHqRKGsjmj7dJynrUCZmwT5RBdGtavw/edit#gid=932531650) and Lisa is tagged to inform her of the changes
- [ ] Images are optimized using ImageOptim
- [ ] Unit tests are added to test behavior
<details><summary>Labels</summary>

- `T:` type of PR
Bug-fix, Chore, Enhancement
- `F:` feature, so that related PRs can easily be grouped
FeatureName
- `D:` relevant platform
Android, iOS
- `Native:` if this PR requires a native release
Android, iOS
- `S:` external deps
Waiting for backend, Waiting for translations
- Helpers
WIP, Direct PR
</details>

- [ ] Labels are added
- [ ] Assigned to a milestone (ask the App Lead if unsure)

## For the reviewers
Copy-paste this in a comment below, post it, and start checking things.

<details>
<summary>Reviewer Comment</summary>

```
### Security
- [ ] If the `eval` function is used, unvalidated user input is not passed to the function (Code Injection).
- [ ] I can verify the new dependency added in the `package.json` does not seem malicious.
### Review
- [ ] I understand the purpose of this PR
- [ ] I understand the code of this PR
- [ ] Target branch is correct(`develop`, `release/alpha`, `release/beta` or `release/prod`)
- [ ] Works correctly on Android 5 and higher
- [ ] Works correctly on iOS
- [ ] The code quality is high (proper location, proper structure, good names...)
- [ ] There's enough documentation to explain what is going on.
#### Just before merging
- [ ] All my requested changes have been resolved
```

</details>

umarniz commented on android/src/main/java/com/twiliorn/library/TwilioEvent.java:
```diff
@@ -0,0 +1,27 @@
+
+    @Override
+    public void dispatch(RCTEventEmitter rctEventEmitter) {
+        Log.i("TwilioEvent"," "+getViewTag()+" "+eventName+" "+ event);
```
Lets remove the log statement

umarniz reviewed (commented):
@jagamypriera could you make this PR for the up-stream repo: https://github.com/blackuy/react-native-twilio-video-webrtc/pulls

We would want this merged in the original repo as well if it helps more people :)

jagamypriera commented:
Addressed feedback from @umarniz 👍 

This PR moved to [here](https://github.com/senseobservationsystems/react-native-twilio-video-webrtc/pull/20) because of wrong branch.
